sdk/src/main/java/io/customer/sdk/extensions/RandomExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package io.customer.sdk.utils
+
+import java.util.*
+
+/**
+ * Set of extensions for primitive types to get a random value. Used for tests.
+ */
+
+fun String.Companion.random(length: Int): String {
```

Comment:
Any specific reason for using `String.Companion.random` instead of `String.random`, apart from dealing with `char` and `char arrays`?

Reply:
Using `String.Companion.random` means I can do: `String.random` to get a random string. 

Using `String.random` means I have to call: `"".random` to get a random string. 

Having a static property (via `String.Companion`) makes more sense to me from the code above.